index.html

```diff
@@ -35,5 +35,156 @@ <h2>I'm a Software Developer</h2>
             </ul>
         </div>        
     </section>
+    <section class="works">
+        <div class="card-1">
+            <span>My Recent works</span>
+        </div>
+        <div class="card-2">
+            <img src="/images/gym-image.png" alt="Placeholder">
+            <p id="story">Multi-Post Stories</p>
```

Comment:
- [ ]  I will recommend you ensure that the title of your project cards are wrapped in heading tags e.g `(<h3>`) and not a `<p>` tag because it represents the title of a project and as such the heading semantic tag should be used so that screen readers and browsers will be able to understand what every part of your website represents 👍🏽